src/main/java/io/github/ascopes/protobufmavenplugin/AbstractGenerateMojo.java

```diff
@@ -101,6 +102,21 @@ public abstract class AbstractGenerateMojo extends AbstractMojo {
   @Parameter
   private @Nullable List<File> sourceDirectories;
 
+  /**
+   * The scope to resolve dependencies with.
```

Comment:
Use a different name for this feature so it doesn't get confused with actual dependency scopes (compile, provided, system, test, etc)